src/components/LoansByCategory/QuickFilters/QuickFilters.vue

```diff
@@ -187,7 +187,22 @@ export default {
 		loanChannelQueryMapMixin
 	],
 	watch: {
-		selectedCategory() {
+		selectedCategory(categoryId) {
+			const catId = Number(categoryId);
+			const queryMap = loanChannelQueryMapMixin.data().loanChannelQueryMap;
+			const categoryFilter = catId === 0 ? {} : queryMap
+				.find(channel => channel.id === catId)?.flssLoanSearch;
+			// These categories use location/gender/sort by for FLSS and need
+			// to have dropdowns preset
+			if (catId === 5) { // women
```

Comment:
Changing category "selects" other filters in the page, but it doesn't reset others, for example, selecting location, then "women", doesn't reset Location. Is that expected?

Reply:
I'm double checking this with design~

Reply:
I actually ended up changing this to reset!